CMakeLists.txt

```diff
@@ -59,6 +58,14 @@ else()
     set(HAVE_SYMENGINE_LLVM False)
 endif()
 
+if(WITH_NUMPY)
+    find_package(NumPy REQUIRED)
+    include_directories(${NUMPY_INCLUDE_PATH})
```

Comment:
I just checked and there are no symbols from numpy libraries linked in. It seems we are not using anything that requires linking. I wonder whether that means we can build with one numpy version and have it work on other numpy versions.